test/index.test.ts

```diff
@@ -52,8 +52,10 @@ describe('monika', () => {
     .do(() =>
       cmd.run(['--config', resolve('./test/testConfigs/noInterval.yml')])
     )
-    .it('runs with config without interval', (ctx) => {
-      expect(ctx.stdout).to.contain('Starting Monika.')
+    .catch((error) => {
+      expect(error.message).to.contain(
+        'The total timeout values of all requests in the probe with name "Example" should be less than 10 seconds. Current interval value is 10 seconds (default value) but the timeout values of this probe is 10000 seconds. To fix this, define the interval value to be greater than 10 seconds or reduce the timeout values of each requests so that the sum of all timeout is less than 10 seconds.'
```

Comment:
I don't understand how this test can ever pass. It has the words `(default value)`, but those words are nowhere to be found in the main code. Not even in the src/components/config/validate.ts file. 

What did I miss?

Reply:
Updated!

Reply:
Sorry that's an invalid test since the Monika doesn't throw any error the test runner doesn't catch anything and skips the test. I will remove the test :)